x-pack/plugins/cloud_integrations/cloud_experiments/public/launch_darkly_client/launch_darkly_client.ts

```diff
@@ -22,46 +23,81 @@ export interface LaunchDarklyUserMetadata
 }
 
 export class LaunchDarklyClient {
-  private launchDarklyClient?: LDClient;
+  private initiated = false;
+  private canceled = false;
+  private launchDarklyClient = new BehaviorSubject<LDClient | null>(null);
```

Comment:
Nit: we tend to name RxJs subscribables with a trailing $.

Should we name this `_launchDarklyClient$` and remove the one 2 lines below?

Being private, I don't think that we need the protection of exposing it `.asObservable()`.

WDYT?

Reply:
Good point. 👍  Addressed in https://github.com/elastic/kibana/pull/181124/commits/d9eb81b2a6ae5983614ad9ae9cc30e9f820f2e38